BackendTaskPilot/BackendTaskPilot.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
With net7.0 being "end of support", it would be good practice to update to the latest patch of net8.0.